### PR TITLE
Implement All Contributors Badge, run [GitHubAllContributors]

### PR DESCRIPTION
**Pull request by JoeIzzard**
Implements #5161

Adds the 'all contributors' badge, using the contributor count color scale.

### CODE REVIEW

File: services/github/github-all-contributors.service.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const { renderContributorBadge } = require('../contributor-count')
+const { ConditionalGithubAuthV3Service } = require('./github-auth-service')
+const { fetchJsonFromRepo } = require('./github-common-fetch')
+const { documentation } = require('./github-helpers')
+
+const schema = Joi.object({
+  contributors: Joi.array(),
```
Comment (calebcartwright) [inline services/github/github-all-contributors.service.js:10]: We need to ensure that the contributors array always exists as part of the response validation, so let's add a `.required()`
```suggestion
  contributors: Joi.array().required(),
```

**Comment by JoeIzzard** [comment]
Thanks for pointing that out @calebcartwright, I thought the required at the end of the schema did that, but wasn't sure. I've implemented that change 👍

File: services/github/github-all-contributors.service.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+module.exports = class GithubAllContributorsService extends ConditionalGithubAuthV3Service {
+  static get category() {
+    return 'activity'
+  }
+
+  static get route() {
+    return {
+      base: 'github/all-contributors',
+      pattern: ':user/:repo',
```
Comment (calebcartwright) [inline services/github/github-all-contributors.service.js:21]: Sorry one more question/thought regarding the branch. In the referenced issue you mentioned:

> As the Bot updates the master branch,

Is that always explicitly master, or is it the default branch in the repository?

Comment (JoeIzzard) [reply services/github/github-all-contributors.service.js:21]: Just done some testing, it does indeed push to the default branch not always master. I will look into adding a way to handle that, and a specified branch.

Comment (calebcartwright) [reply services/github/github-all-contributors.service.js:21]: We can just add the `branch` route param, looks like you've already got everything else wired up to handle it :+1: 

```suggestion
      pattern: ':user/:repo/:branch*',
```

Comment (JoeIzzard) [reply services/github/github-all-contributors.service.js]: I've implemented that, as well as update the example given to the Frontend, from the searching I have done it seems there's currently no way to use the default branch (Issue #4114) if a branch isn't specified, would the be correct?

Comment (calebcartwright) [reply services/github/github-all-contributors.service.js]: Correct. We do the best we can today, and for folks with different defaults they need to include an extra param in their badge url

**Pull request merged**